### ISSUE TITLE
Pack UI: Update after install

### ISSUE THREAD

**Issue by christopherjbaker** [StackStorm/st2web#467]
After installing a pack, I have to refresh to be able to configure it. Since the pack installation is updated live, would be nice if the sidebar options to then configure the pack were updated live as well.

**Comment by LindsayHill** [comment]
Agreed. Current status (with 2.8dev) is that the pack moves to "Installing" as soon as I click Install, but it never moves to "Installed" when the action has completed.
